test/test_basic.py

```diff
@@ -71,133 +62,117 @@ async def _call():
 
     def test_noarg_noret_sync(self):
         self.add_method('', 'Do', '', '', '')
-        message = self.build_call('Do')
-        self.assertEqual(self.bus_user.call(message).get_body(), ())
+        result = self.bus_user.call_method(N_TEST, '/', I_TEST, 'Do')
```

Comment:
I actually sort of liked that your wrapper filled this in a bit.  I wonder if we could figure out some way to save this stuff using a partial application or so...

Alternatively, maybe you could make it "nicer" by doing something evil like

```
TEST_ADDR = ['org.freedesktop.Test', '/', 'org.freedesktop.Test.Main']
```

and then

```
self.bus_user.call_method(*TEST_ADDR, 'Do')
```

Reply:
You are right, this looks nicer. How about now?

Reply:
:+1: